### PR TITLE
BOAC-1781 Enable attachment download links

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.ets.berkeley.edu/jira/browse/BOAC-1781

### CODE REVIEW

File: src/components/note/AdvisingNote.vue [hunk 1]
```diff
@@ -40,8 +40,23 @@
     <div v-if="size(note.attachments)">
       <div class="pill-list-header mt-3 mb-1">{{ size(note.attachments) === 1 ? 'Attachment' : 'Attachments' }}</div>
       <ul class="pill-list pl-0">
-        <li v-for="(attachment, index) in note.attachments" :key="attachment" class="mt-2">
-          <a :id="`note-${note.id}-attachment-${index}`" href="#" class="pill text-nowrap"><i class="fas fa-paperclip"></i> {{ attachment }}</a>
+        <li
+          v-for="(attachment, index) in note.attachments"
+          :key="attachment"
+          class="mt-2"
+          @click.stop>
```
Comment (paulkerschen) [inline src/components/note/AdvisingNote.vue]: This line mitigates the annoyance of https://jira.ets.berkeley.edu/jira/browse/BOAC-1875 by keeping the note open when an attachment is clicked.

File: src/components/note/AdvisingNote.vue [hunk 2]
```diff
@@ -65,6 +81,11 @@ export default {
     allUsers: undefined,
     author: undefined
   }),
+  computed: {
+    isPreCsNote() {
+      return this.get(this.note, 'createdBy') === 'UCBCONVERSION';
```
Comment (johncrossman) [inline src/components/note/AdvisingNote.vue]: @paulkerschen Minor: `isPreCsNote` can be set in `created()` – does not need to be `computed`?

**Comment by johncrossman** [comment]
Looks good @paulkerschen  Posted a minor concern.

**Comment by johncrossman** [comment]
We can have follow up PR on `computed`, if so desired.

**Pull request merged**